PHPCI/Model/Build/GitlabBuild.php

```diff
@@ -47,7 +47,8 @@ public function getFileLinkTemplate()
             'http://%s/%s/blob/%s/{FILE}#L{LINE}',
             $this->getProject()->getAccessInformation("domain"),
             $this->getProject()->getReference(),
-            $this->getBranch()
+            $this->getgetCommitId()
```

Comment:
you're right!! Thanks! I edited directly on github, in my local install it's ok..
Saluti dal Messico!